specs/Terminate Spec.md

```diff
@@ -0,0 +1,60 @@
+# **Terminate Program Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Shortcut for users to quickly terminate a running program
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+## 1. Overview
+
+**1.1. Elevator Pitch / Narrative** 
+
+Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike can completely shutdown the app quicker using a Keyboard shortcut if he has this PowerToy installed on his computer.
```

Comment:
Is speed really the point of this? I thought it was to provide a visual and accessible last-resort mechanism to eliminating a process that can't otherwise be removed by traditional/safe means.